src/applications/coronavirus-chatbot/index.js

```diff
@@ -1,4 +1,5 @@
-import 'botframework-webchat';
```

Comment:
Context: The NPM module was different from the version used in the working prototype from the CDN. It might be worth investigating why this doesn't work later, but for now, the plan is to add a `<script>` tag to the `vagov-content` page pointing to the working version in the CDN.